main.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat Alternates', sans-serif;
+  max-width: 768px;
+  border: 1px solid #000;
+  height: 1024px;
+  margin: auto;
+  background: radial-gradient(ellipse at center, rgba(56, 185, 242, 0.8) 0%,rgba(57,138,187,0.8) 100%);
+  font-size: 1rem;
+}
+.welcome {
+  display: flex;
+  flex-direction: column;
+  margin-top: 2rem;
+  color: #fff;
+  padding: 0px 100px;
+}
+.welcome__title {
+  font-family: 'Henny Penny', cursive;
+  font-size: 4rem;
+  margin: auto;
+  letter-spacing: 0.2rem;
+}
+.welcome__title__organization {
+  text-align: center;
+  display: block;
+  margin-top: -45px;
+}
+.welcome__image {
+  margin: 25px auto;
+  height: 500px;
```

Comment:
No, this web will be build tablet first, as this is the technology most commonly used by the therapists at CAMHS.